src/amo/utils/server.js

```diff
@@ -59,3 +67,23 @@ export const viewFrontendVersionHandler = ({
     });
   };
 };
+
+export const viewHeartbeatHandler = ({
+  _config = config,
+  _fetch = fetch,
+}: ViewHeartbeatHandlerParams = {}): ExpressHandler => {
+  const apiURL = `${_config.get('apiHost')}${_config.get(
+    'apiPath',
+  )}${_config.get('apiVersion')}/site/?disable_caching`;
+  return async (req: typeof $Request, res: typeof $Response) => {
+    const response = await _fetch(apiURL);
+    if (response.status !== 200) {
+      res.sendStatus(400);
```

Comment:
Is this the right status code to return?

Reply:
I just assumed that anything other than 200 would suffice, but it's a good question. @bqbn Is there a preferred status code to return when things are *not* ok?

Reply:
the document says "Should return a 200 if the service is healthy, and a 500 otherwise. "